src/EnrollChoicesController.js

```diff
@@ -100,11 +100,13 @@ function (Okta, FormController, Enums, RouterUtil, FactorList,
         case Enums.ALL_OPTIONAL_SOME_ENROLLED:
         case Enums.ALL_OPTIONAL_NONE_ENROLLED:
           var enrolled = factors.where({ enrolled: true }),
-              notEnrolled = factors.where({ enrolled: false }),
+              notEnrolled = factors.filter(function (m) {
```

Comment:
Can you change `m` to something more descriptive like `factor`? Also maybe we should change the comment because it does not make sense with this code change now..